### PR TITLE
Modify dropzones css to take it into account of themes fixes #12924

### LINKED ISSUE #12924: [frio] new upload area in post composer should take dark and black color scheme into account
The new upload area does not take dark and black mode into account. Would be nice to have the new area reflecting the selected color scheme.

upload

### PR DESCRIPTION
**Pull request by xundeenergie**
I modified the dropzone-css and the place where it will be loaded to make dropzone better configureable with color-schemes and css-variables.

It fixes https://github.com/friendica/friendica/issues/12924

### CODE REVIEW

File: view/theme/frio/css/dropzone.frio.css
Comment (MrPetovan) [inline view/theme/frio/css/dropzone.frio.css:1]: The indentation changes to this file look wrong, only the first level was changed. Please have the indentation consistent through the file.

File: view/theme/frio/templates/head.tpl [hunk 1]
```diff
@@ -55,8 +55,8 @@
 	media="screen" />
 <link rel="stylesheet" href="view/theme/frio/css/font-awesome.custom.css?v={{$smarty.const.FRIENDICA_VERSION}}"
 	type="text/css" media="screen" />
-<link rel="stylesheet" href="view/theme/frio/css/dropzone.min.frio.css?v={{$smarty.const.FRIENDICA_VERSION}}"
-	type="text/css" media="screen" />
+<!--link rel="stylesheet" href="view/theme/frio/css/dropzone.frio.css?v={{$smarty.const.FRIENDICA_VERSION}}"
```
Comment (MrPetovan) [inline view/theme/frio/templates/head.tpl]: Please remove commented out code before creating a PR.

**Review by MrPetovan — CHANGES_REQUESTED** [review]

File: view/theme/frio/css/dropzone.frio.css
Comment (xundeenergie) [reply view/theme/frio/css/dropzone.frio.css:1]: now it should be fixed.

How can i check this myself? 
My experiences with woodpecker are almost zero...

**Comment by xundeenergie** [comment]
In the conflicting file, the "color: white;" must be removed!

**Comment by MrPetovan** [comment]
To fix the conflict, rebase your branch on the latest 2023.03-rc, then edit the file to fix the conflict and continue the rebase as indicated in `git status`.

**Comment by MrPetovan** [comment]
And now you need to do it again since I merged the same PR for the minified file. 😅

**Comment by xundeenergie** [comment]
okeeeee :D

It's rebased again.

**Review by MrPetovan — APPROVED** [review]

**Pull request merged**